go/okapi/DidComm.go

```diff
@@ -4,13 +4,15 @@ import (
 	"github.com/trinsic-id/okapi/go/okapiproto"
 )
 
+// DidCommer implements the DIDComm Messaging protocol
```

Comment:
fix 3e, otherwise Go gives us errors.  (e.g. [here](https://github.com/trinsic-id/okapi/runs/5156299573?check_suite_focus=true), view the "Build and Test Golang" output)